scenario/stages/1_checkcloud.yaml

```diff
@@ -5,9 +5,11 @@
 
 preparation:
   - create_snapshot: True
+  - create_image_snapshot_dst: True
```

Comment:
Image snapshot doesn't look as required here. Actually I think this whole scenario should only include `CheckCloud` step:
```
preparation:
   - act_check_dst_cloud: True

rollback:

process:
```